files/images/teamspeak_viewer/badges/gamescom_2016_details.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64"><title>gamescom-2016-details-badge</title><g id="Ebene_1" data-name="Ebene 1"><path d="M20.54,36.63,25.9,8.57h.39l4.28,22.85,1.22-3.08h.36l1.28,3L37.75,8.57h.39l5.33,28V48.4h-.2l-22.54-.1h-.2V36.63Zm3.28.16V45l16.38.07V36.77L38,24.1,35.62,37.83l0,.16H33.37l0-.13L32,34l-1.59,4,0,.12h-2l0-.16L26,24.13Z" style="fill:#8190aa;fill-rule:evenodd"/><path d="M7.52,6.11A53.85,53.85,0,0,1,32,0,54.59,54.59,0,0,1,56.47,6.11l.93.46v31l-.06.43a41.44,41.44,0,0,1-9.93,17.86C42.43,61.06,36.65,64,32,64s-10.19-3.39-15.08-8.5A42.23,42.23,0,0,1,6.66,38l-.06-.43v-31l.92-.46ZM32,3.33A50.18,50.18,0,0,0,9.95,8.64V37.36a38.93,38.93,0,0,0,9.37,15.83c4.3,4.49,9.09,7.47,12.68,7.47,3.79,0,8.66-2.57,13-7.1a38,38,0,0,0,9.05-16.19V8.65A50.87,50.87,0,0,0,32,3.33Z" style="fill:#8190aa;fill-rule:evenodd"/><path d="M19.19,25.55H18.07v2.9c-2.15-2-3.23-2.87-4.46-2.87A2.21,2.21,0,0,0,11.36,28a2.91,2.91,0,0,0,1.25,2.45l.8-.79A1.91,1.91,0,0,1,12.49,28a1.08,1.08,0,0,1,1.12-1.13c1,0,1.76.74,4.53,3.43h1Z" style="fill:#8190aa;fill-rule:evenodd"/><path d="M19.33,21.91c0-1.88-2.06-2.61-4-2.61s-4,.74-4,2.61,2,2.6,4,2.6,4-.73,4-2.6Zm-1.13,0c0,1-1.35,1.3-2.86,1.3s-2.84-.28-2.84-1.3,1.33-1.31,2.84-1.31S18.19,20.88,18.19,21.91Z" style="fill:#8190aa;fill-rule:evenodd"/><polygon points="19.19 15.53 11.47 15.53 11.47 16.65 13.59 18.73 14.37 17.98 13.13 16.81 19.19 16.81 19.19 15.53" style="fill:#8190aa;fill-rule:evenodd"/><path d="M19.33,11.65a2.4,2.4,0,0,0-2.58-2.39c-1.64,0-2.49,1-2.49,2.1A2,2,0,0,0,15.31,13h-.19c-1.33,0-2.64-.53-2.64-1.61a1.4,1.4,0,0,1,.68-1.24l-.93-.65a2.34,2.34,0,0,0-.89,1.9c0,1.94,1.78,2.89,4,2.89s4-.72,4-2.68Zm-1.13.07c0,1-1.08,1.26-1.87,1.31a1.56,1.56,0,0,1-1-1.28c0-.64.39-1.19,1.43-1.19A1.26,1.26,0,0,1,18.19,11.72Z" style="fill:#8190aa;fill-rule:evenodd"/><rect y="63.83" width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect x="63.83" y="63.83" width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect x="63.83" width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect x="63.83" width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/></g></svg>
```

Comment:
Why are there no line breaks here?

Reply:
Rechecked with original source. The file is untouched from original.